### DISCUSSION TITLE
blog/understand-istio-vs-and-dr/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#311] [Announcements]
# blog/understand-istio-vs-and-dr/

本文将 Istio 中的流量治理与现实世界中的交通管理类比，可以帮助你快速理解 VirtualService 和 DestinationRule 设置的功能。

https://jimmysong.io/blog/understand-istio-vs-and-dr/

**Comment by rootsongjc** [comment]
来自朋友圈的评论：VirtualService 大部分对应的是 Route，DestinationRule 大部分对应 Cluster。
